registry/storage/driver/s3-aws/s3.go

```diff
@@ -1350,7 +1351,7 @@ type writer struct {
 	cancelled bool
 }
 
-func (d *driver) newWriter(key, uploadID string, parts []*s3.Part) storagedriver.FileWriter {
+func (d *driver) newWriter(ctx context.Context, key, uploadID string, parts []*s3.Part) storagedriver.FileWriter {
```

Comment:
You add the ctx here but never actually use it

Reply:
Same comment as above.

Reply:
Actual facepalm 🤦‍♂️ nice spot @Jamstah - I misread the comment

Reply:
fixed: https://github.com/distribution/distribution/pull/4109/commits/52d047913d5032bebed56670a873ce42f5b6c1cd